enterprise/frontend/src/embedding-sdk/store/reducer.ts

```diff
@@ -113,31 +85,31 @@ export const sdk = createReducer(initialState, builder => {
     token: { ...state.token, loading: true },
   }));
 
-  builder.addCase(refreshTokenAsync.fulfilled, (state, action) => ({
-    ...state,
-    token: {
-      ...state.token,
```

Comment:
nice! removing this spread is great because it was misleading as this object only has 3 properties, and we define all of them already.